packages/core/src/controllers/ConnectorController.ts

```diff
@@ -280,8 +281,23 @@ export const ConnectorController = {
     state.connectors = this.getConnectors(namespace)
   },
 
+  /**
+   * Clears the filter by namespace and updates the connectors.
+   * If the current filterByNamespace has a value, it will fetch the recommended wallets with all requested chains.
+   */
   clearNamespaceFilter() {
     state.filterByNamespace = undefined
     state.connectors = this.getConnectors()
+  },
+
+  getIsConnectedWithWC() {
+    const chains = Array.from(ChainController.state.chains.values())
+    const isConnectedWithWC = chains.some(chain => {
+      const connectorId = StorageUtil.getConnectedConnectorId(chain.namespace)
+
+      return connectorId === ConstantsUtil.CONNECTOR_ID.WALLET_CONNECT
+    })
+
+    return isConnectedWithWC
```

Comment:
why is this here? it's not really using anything on this controller's state, should probably be a util